tests/e2e/conftest.py

```
@@ -484,6 +484,37 @@ def ironclaw_reborn_binary():
+        print("Building Reborn ironclaw with E2E test support (this may take a while)...")
+        # Cargo fingerprints feature sets separately but publishes both at
+        # the same top-level path. Removing that generated output forces Cargo
+        # to materialize the requested feature variant before we copy it.
+        cargo_binary.unlink(missing_ok=True)
+        subprocess.run(
+            [
+                "cargo", "build",
+                "-p", "ironclaw",
+                "--bin", "ironclaw",
+                "--features", "e2e-test-support",
+            ],
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)